framework/src/errors.ts

```diff
@@ -15,6 +15,7 @@
 
 export class FrameworkError extends Error {
 	public name: string;
```

Comment:
We didn't set this value in the constructor, super will set it to `Error`. Need to explicitly set `this.name = 'FrameworkError';` in the constructor.

Reply:
it is actually already set in the constructor